image/base/Dockerfile

```diff
@@ -36,7 +51,13 @@ RUN --mount=type=tmpfs,target=/var/log \
     libreadline-dev anacron wget \
     psmisc whois brotli libunwind-dev \
     libtcmalloc-minimal4 cmake \
-    pngcrush pngquant ripgrep poppler-utils; \
+    pngcrush pngquant ripgrep poppler-utils \
+# imagemagick runtime dependencies
+    libheif1 libjbig0 libtiff6 libpng16-16 libfontconfig1 \
+    libwebpdemux2 libwebpmux3 libxext6 librsvg2-2 libgomp1 \
+    fonts-urw-base35 \
```

Comment:
We are duplicating the required runtime dependencies between the `Dockerfile` and the `install-imagemagick` script. Feels like it will be quite brittle.

Reply:
Also wondering... is there a way to statically compile imagick so we don't have to think about this?

Reply:
install-imagemagick includes static libs and -dev libs as well which are only needed to compile - this is only a subset or a list of non-dev .so libs that magick needs at runtime.

I looked into an LD -static flag, but that felt like a whole other rabbit hole. The sources I've seen so far point to the compilation being able to not compile *more* dynamic libs so the actual magick binary is built statically, but sources I've found are if you need to enable extensions as we do, we still are depending on these libs.